lib/membrane/children_spec.ex

```diff
@@ -318,7 +318,7 @@ defmodule Membrane.ChildrenSpec do
 
   @type children_spec_options :: [
           group: Child.group(),
-          crash_group_mode: Membrane.CrashGroup.mode() | nil,
+          crash_group_mode: Membrane.CrashGroup.mode(),
```

Comment:
```suggestion
          crash_group_mode: :temporary | nil,
```

and we can remove `Membrane.CrashGroup` altogether.

Reply:
It is used in 2 other places as well, so I can remove this `CrashGroup` module and move typespec to the another file, but I would still have a typespec describing `:temporary | nil`